website/common/locales/en/communityGuidelines.json

```diff
@@ -97,35 +99,30 @@
   "commGuideList11A": "Reminders of Public Space Guidelines",
   "commGuideList11B": "Warnings",
   "commGuideList11C": "Requests",
-  "commGuideList11D": "Deletions (Mods/Staff may delete problematic content)",
-  "commGuideList11E": "Edits (Mods/Staff may edit problematic content)",
+  "commGuideList11D": "Deletions (Staff may delete problematic content)",
+  "commGuideList11E": "Edits (Staff may edit problematic content)",
 
   "commGuideHeadingRestoration": "Restoration",
   "commGuidePara061": "Habitica is a land devoted to self-improvement, and we believe in second chances. <strong>If you commit an infraction and receive a consequence, view it as a chance to evaluate your actions and strive to be a better member of the community</strong>.",
   "commGuidePara062": "The announcement, message, and/or email that you receive explaining the consequences of your actions is a good source of information. Cooperate with any restrictions which have been imposed, and endeavor to meet the requirements to have any penalties lifted.",
-  "commGuidePara063": "If you do not understand your consequences, or the nature of your infraction, ask the Staff/Moderators for help so you can avoid committing infractions in the future. If you feel a particular decision was unfair, you can contact the staff to discuss it at <a href='mailto:admin@habitica.com' target='_blank'>admin@habitica.com</a>.",
+  "commGuidePara063": "If you do not understand your consequences, or the nature of your infraction, ask the Staff for help so you can avoid committing infractions in the future. If you feel a particular decision was unfair, you can contact the staff to discuss it at <a href='mailto:admin@habitica.com' target='_blank'>admin@habitica.com</a>.",
 
-  "commGuideHeadingMeet": "Meet the Staff and Mods!",
-  "commGuidePara006": "Habitica has some tireless knights-errant who join forces with the staff members to keep the community calm, contented, and free of trolls. Each has a specific domain, but will sometimes be called to serve in other social spheres.",
-  "commGuidePara007": "Staff have purple tags marked with crowns. Their title is \"Heroic\".",
-  "commGuidePara008": "Mods have dark blue tags marked with stars. Their title is \"Guardian\".",
+  "commGuideHeadingMeet": "Meet the Staff",
+  "commGuidePara007": "The Habitica Staff keep the app and sites running and can act as chat moderators. They have purple tags marked with crowns. Their title is \"Heroic\".",
   "commGuidePara009": "The current Staff Members are (from left to right):",
   "commGuideAKA": "<%= habitName %> aka <%= realName %>",
   "commGuideOnGitHub": "<%= gitHubName %> on GitHub",
-  "commGuidePara010": "There are also several Moderators who assist the staff members. They were selected carefully, so please give them your respect and listen to their suggestions.",
-  "commGuidePara011": "The current Moderators are (from left to right):",
+  "commGuideStaffRole": "- <%= role %>",
```

Comment:
I don't think it's useful to create a string token here. You're asking translators to translate a single hyphen..?